meta-balena-common/recipes-bsp/efitools/efitools/0001-hash-to-efi-sig-list.c-Allow-raw-hashes-as-arguments.patch

```diff
@@ -0,0 +1,74 @@
+From cd50901c9b983d3b06e598e276d848a707ab0b6e Mon Sep 17 00:00:00 2001
```

Comment:
It would be good to submit this upstream, even as an RFC for feedback on the argument ambiguity. I'd like to see it merged, if possible.